10.0/base_requirements.txt

```diff
@@ -67,6 +67,7 @@ anthem==0.12.0
 # test / lint
 flake8
 pytest
+pluggy
```

Comment:
for our future selves, I'd add a comment on top of it saying: keep this unpinned to satisfy `pytest`

Reply:
I'll add a comment on the # test / lint line why it is unpinned, everything in this section is under the same rule.

Reply:
@simahawk fixed